_health-care/_js/_components/_health-care-app.jsx

```diff
@@ -28,6 +28,17 @@ class HealthCareApp extends React.Component {
         year: "2014",
       };
     }
+    if (this.state.applicationData.fullName === undefined) {
```

Comment:
It's actually a guard against a stale localStorage object, but yes...mostly for testing.